Classes/Manager/FLEXManager.m

```diff
@@ -288,6 +290,15 @@ - (void)registerGlobalEntryWithName:(NSString *)entryName viewControllerFutureBl
     [self.userGlobalEntries addObject:entry];
 }
 
+- (void)setCustomViewerForContentType:(NSString *)contentType viewControllerFutureBlock:(FLEXCustomContentViewerFuture)viewControllerFutureBlock
+{
+    NSParameterAssert(contentType);
```

Comment:
Nitpicky, but if we're checking for it at all, might as well check for non-empty strings too:
```suggestion
    NSParameterAssert(contentType.length);
```